teaser/data/output/modelicatemplate/AixLib/AixLib_Multizone

```diff
@@ -3,6 +3,7 @@ within ${bldg.parent.name}.${bldg.name};
 model ${bldg.name}
   "This is the simulation model of ${bldg.name} with traceable ID ${bldg.building_id}"
 
+% if bldg.internal_gains_mode==1 or bldg.internal_gains_mode==2:
```

Comment:
In general I find it better to use strings to decide something like this instead of integers. BUt we can also do this as a new feature (and then for whole teaser)